Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -605,7 +613,10 @@ protected void Delete(OperationContext context, ShortHash hash)
             Store(context, hash, entry: null);
         }
 
-        private Task ForceCacheFlushAsync(OperationContext context)
+        /// <summary>
+        /// Should only be called from <see cref="ForceCacheFlush(ContentStore.Tracing.Internal.OperationContext, ContentLocationDatabaseCounters?, bool)"/>

```

Comment:
Maybe instead of making this comment, this method should be made internal to avoid any other options?